lib/mapper.js

```diff
@@ -19,7 +19,7 @@ var time = require('unix-time');
 exports.track = function(msg, settings){
   var eventName = msg.event();
   var appEvents = settings.appEvents;
-  if (appEvents && appEvents[eventName]) {
+  if (appEvents && appEvents[eventName] && (appEvents[eventName]!=="null") ) {
```

Comment:
Not sure I understand this yet. Can you send a screenshot of the settings in the Segment UI when `appEvents[eventName]!=="null"` is true?